pkg/cloud-controller-manager/loadBalancer.go

```diff
@@ -68,6 +70,10 @@ func (l *LoadBalancerManager) GetLoadBalancer(ctx context.Context, clusterName s
 }
 
 func (l *LoadBalancerManager) GetLoadBalancerName(ctx context.Context, clusterName string, service *v1.Service) string {
+	// If users don't specify cluster name by the command flag, we adopt the cluster name from the cloud config file
+	if clusterName == "" || clusterName == "kubernetes" {
```

Comment:
If users don't specify cluster name by the command flag, we adopt the cluster name from the cloud-config file.